roles/parallels/tasks/install.yml

```diff
@@ -124,6 +124,12 @@
     group: staff
     mode: '0644'
 
+- name: Ensure the Parallels Desktop app is not running
+  become: no
+  parallels_desktop:
+    state: >-
+      {{ parallels_max_termination_severity }}
```

Comment:
What's the advantage of folding and chomping here? Something more than avoid the need for quotes?

Reply:
Yeah, I suppose that's my personal style preference. Indeed, I tend to default to `>-` in YAML to avoid quoting. This let's me stop thinking of whether I'm going to introduce a syntax error/typo and seems like a clean approach. Moreover, if there's a need to use quotes inside the mustache, I don't need to escape anything. Also, if I ever need to make the jinja2 explession more complex, it allows me to multi-line chunks with filter pipes without having to inspect it for the need to drop quoting first — this contributes to cleaner diffs as well.